src/app/legacy/components/ScrollablePromo/index.jsx

```diff
@@ -35,10 +42,50 @@ const PromoWrapper = styled.div`
   }
 `;
 
-const LabelComponent = styled.strong`
-  display: block;
-  ${({ script }) => script && getDoublePica(script)};
+const ScrollablePromoContainer = styled.div`
+  background: ${GREY_2};
+  padding: ${GEL_SPACING};
+  display: flex;
+  overflow-x: auto;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+  &::-webkit-scrollbar {
+    display: none;
+  }
+  ${({ experimentVariant }) =>
+    experimentVariant &&
+    experimentVariant !== 'none' &&
+    `
+    padding: 0 ${GEL_SPACING} ${GEL_SPACING_DBL};
+    margin: 0rem;
+
+    @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+      padding: 0 ${GEL_SPACING_DBL} ${GEL_SPACING_DBL};
+      margin: 0 -0.2rem;
+    }
+    
+    @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+      margin: 0 -0.8rem;
+    }
+    
+    @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+      display: none;
+    }
+
+    width: 100vw;
+  `}
+`;
+
+const LabelComponent = styled(({ ariaLabel, experimentVariant, ...props }) => (
+  <strong aria-label={ariaLabel} {...props} />
```

Comment:
Is this aria-label for the experiement version of the component?

Reply:
I have now separated out a LabelComponent for the original scrollable promo and one for the experiment one, as it is easier than the convoluted conditionals within the one component where something can be missed (like the area label or display type). The side effect of this is that it has also addressed the issue in your comment above this one! Now the original component should be unchanged and so not need testing.